pos_order_load/i18n/fr.po

```diff
@@ -0,0 +1,145 @@
+# Translation of Odoo Server.
```

Comment:
Currently there's no need for the pot file because is generated and exported to Transifex automatically. Actually, it's not recommended to export it because it can confuse the user about what is expected to be exported to Transifex.